src/Nethermind/Nethermind.Blockchain.Test/BlockTreeTests.cs

```diff
@@ -1800,8 +1800,6 @@ public void BlockAddedToMain_should_have_updated_Head()
             Block block0 = Build.A.Block.WithNumber(0).WithDifficulty(1).TestObject;
             Block block1 = Build.A.Block.WithNumber(1).WithDifficulty(2).WithParent(block0).TestObject;
             AddToMain(blockTree, block0);
-            blockTree.SuggestBlock(block0);
-            blockTree.UpdateMainChain(new[] { block0 }, true);
```

Comment:
Is this not needed?

Reply:
No, because AddToMain(blockTree, block0) does the exact same thing. Its code duplication.